src/main/java/com/boot/product/model/Characteristics.java

```diff
@@ -0,0 +1,82 @@
+package com.boot.product.model;
+
+
+import com.boot.product.dto.CharacteristicsDTO;
+import com.boot.product.dto.ProductDTO;
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+@Accessors(chain = true)
+@Entity
+@Table(name = "characteristics")
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class,property = "id")
+public class Characteristics implements Serializable {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 2L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column
+    private Long id;
+
+    @Column
+    private String shape;
+
+    @Column
+    private String material;
+
+    @Column
+    private String woodEssence;
+
+    @Column
+    private String length;
+
+    @Column
+    private String width;
+
+    @Column
+    private String weight;
+
+    @Column
+    private String characteristic;
+
+    @OneToOne( cascade = CascadeType.ALL)
+    @JoinColumn(name = "product_id", referencedColumnName = "id")
+    private Product product;
+
+
+    public static CharacteristicsDTO characteristicsEntityToDto(Characteristics characteristics) {
```

Comment:
you could use ModelMapper instead of this methods

Reply:
im not shure how to use the model mapper 😄

Reply:
you can follow how it's implemented in other services. it's very straightforward.